R/api-update-news.R

```diff
@@ -10,7 +10,7 @@ update_news <- function(messages = NULL) {
   if (is.null(messages)) {
     messages <- get_top_level_commits(since = get_last_tag()$commit)$message
   } else if (is.list(messages)) {
-    deprecate_stop("0.0.5", "fledge::update_news(range = )",
+    lifecycle::deprecate_stop("0.0.5", "fledge::update_news(range = )",
```

Comment:
I removed the blanket import of lifecycle because there was the warning


```r
* checking whether package ‘fledge’ can be installed ... WARNING
Warning: Found the following significant warnings:
  Warning: replacing previous import ‘lifecycle::last_warnings’ by ‘rlang::last_warnings’ when loading ‘fledge’
See ‘/Users/runner/work/fledge/fledge/check/fledge.Rcheck/00install.out’ for details.
```